evolver/calibration/actions.py

```diff
@@ -6,20 +6,40 @@
 
 
 class CalibrationAction(ABC):
-    class UserInput(pydantic.BaseModel): ...
+    def __init__(self, name: str, description: str):
+        self.description = description
+        self.name = name
+
+    class UserInput(pydantic.BaseModel):
+        pass
```

Comment:
After convo with Jamie, this is a nice way to communicate with frontend what params an action expects.